.github/workflows/test.yml

```diff
@@ -39,6 +39,23 @@ jobs:
         run: task build
       - name: Format
         run: task lint
+
+  docs:
+    name: Docs Test
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: "1.20"
+        id: go
+      - name: Install Task
+        uses: arduino/setup-task@v1
+        with:
+          version: "3.x"
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
```

Comment:
separated the docs test because it's annoying not seeing the tests result because the docs check failed. Also, I updated the docs and can't figure out why I'm getting an error

Reply:
I'll make it req. once it's merged.